api/server/sdk/api/api.swagger.json

```diff
@@ -17,7 +17,7 @@
   "paths": {
     "/v1/cloudbackup": {
       "get": {
-        "summary": "Enumerate",
+        "summary": "Erases an alert for a given resource",
```

Comment:
`make docs` seems to have messed this up.

Reply:
`make proto` made these changes, I reverted them. But `make proto` still produces this wrong output.